### DISCUSSION TITLE
how can I specify a custom folder for models , loras, embeddings etc?

### DISCUSSION THREAD

**Discussion by eyeweaver** [Stability-AI/StableSwarmUI#92] [Q&A]
Hello

Kindly, how can I specify a custom folder for models , loras, embeddings etc?
I tried adding the custom paths in the server configuration settings.. but it doesn't seem to be making any difference

**Comment by mcmonkey4eva** [comment]
Under server configuration is the correct way - you have to restart Swarm after changing them (alpha limitation that will hopefully be fixed eventually).
